vxquery-core/src/main/java/org/apache/vxquery/types/TypeUtils.java

```diff
@@ -27,39 +27,45 @@ public static SequenceType createSequenceType(String str) {
     }
 
     public static SequenceType createSequenceType(StaticContext sCtx, String str) {
-        str = str.trim();
+        String s = str.trim();
```

Comment:
No need to change this now, but I would have preferred, if you would have used "str" for the new variable and changed the name of the parameter as that would have made it clearer what actually has changed.